### ISSUE TITLE
What are the possible actions in the policies

### ISSUE THREAD

**Issue by harianus** [balderdashy/sails-docs#596]
I'm trying to setup some policies for my controllers like this
```js
// config/policies.js

module.exports.policies = {
  '*': 'forbidden',

  PlayController: {
    '*': 'forbidden',
    show: true,
  }
};
```

`forbidden` is a policy I created and that one works. But the `show` action does not work. What are the default actions you can use there? I know about `edit`, but what are the others?

I couldn't find it here: [/sails.config/sails.config.policies.md](https://github.com/balderdashy/sails-docs/blob/master/reference/sails.config/sails.config.policies.md)

**Comment by harianus** [comment]
I found some useful information here http://stackoverflow.com/a/19012154/747044

* index
* find
* create
* update
* destroy

But what is the difference between `find` and `index`? I looks like my `index` route (`/plays`) respond to the `find` actions.

**Comment by jub3i** [comment]
I think your problem is that the first `'*': forbidden` goes through forbidden, making everything... forbidden! :) thats why the second `'*' : 'forbidden'` appears to work.

you should set the first `'*' : 'forbidden'` to `'*' : false`, and then it should work as you intended.

dunno what your talking about in your second post.

**Comment by harianus** [comment]
I must say, it is a fast response.

> you should set the first '*' : 'forbidden' to '*' : false, and then it should work as you intended.

That is not the problem, I'll try to explain it better next time.

> dunno what your talking about in your second post.

That was the whole point, it now works with using `find` instead of `show`:

```js
// config/policies.js

module.exports.policies = {
  '*': 'forbidden',

  PlayController: {
    '*': 'forbidden',
    find: true,
  },
};
```

Does it make sense to create a pull request for the docs where we can explain that the actions `index`,
`find`, `create`, `update`, `destroy` also can be used in `policies.js`?

**Comment by jub3i** [comment]
I'm still not entirely sure what you are getting at. Do you mean that by simply renaming the controllers action from `show` to `find` then makes `find` accessible? If this is the case, that may be a bug with the Blueprint API, please pastebin your code for `forbidden` somewhere so we can follow this up.

**Comment by harianus** [comment]
So the default actions people can use there are `find`, `findOne`, `create`, `update`, `destroy`, `populate`, `add` and `remove`.

So should there be a link on the [/concepts/policies](http://sailsjs.org/documentation/concepts/policies) page to [/reference/blueprint-api#?blueprint-actions](http://sailsjs.org/documentation/reference/blueprint-api#?blueprint-actions) with the text that those `blueprint-actions` can be used in `config/policies.js`?

**Comment by sailsbot** [comment]
Thanks for posting, @harianus.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by harianus** [comment]
@sailsbot makes me sad

![](https://media.giphy.com/media/yoJC2Olx0ekMy2nX7W/giphy.gif)
